src/app/tabs/settings-tab.js

```diff
@@ -140,6 +141,10 @@ function loadVersion (version, queryParams, appAPI, el) {
 
     url = location + 'soljson.js'
   } else {
+    if (version.indexOf('soljson') !== 0 || helper.checkSpecialChars(version) != null) {
```

Comment:
why do you compare to `null`?

Reply:
yes I guess `if (version.indexOf('soljson') !== 0 || helper.checkSpecialChars(version))` would have been sufficient...